src/hooks/useParams.ts

```diff
@@ -27,7 +27,7 @@ export const useParams = (searchType: string, defaultSearchtype: string) => {
     history.push({
       location: location.pathname,
       search: `${searchType}=${getParams}`,
-    });
+    } as any);

```

Comment:
I don't understand why this is needed?

Reply:
This was throwing up a type error and I didn't realize what's wrong here when I typed it as `any`. We have a bug in this code. This should be
```js
history.push({
  pathname: location.pathname,
  search: `${searchType}=${getParams}`,
});
```

It seems we're using this whole useParams hook though, so I'll remove this whole file.